master/Makefile

```diff
@@ -1,7 +1,7 @@
 .DEFAULT_GOAL := build
 SHELL := bash
 GENERATED := packaging/LICENSE
-GENERATION_INPUTS = ../schemas/gen.py $(shell find ./pkg/schemas/ -name '*.go') $(shell find ../schemas/expconf -name '*.json')
+GENERATION_INPUTS = ../schemas/gen.py $(shell find ../schemas/expconf -name '*.go') $(shell find ../schemas/expconf -name '*.json')
```

Comment:
on a sidenote I think this could be done with a single find (if we need both .go and .json) `find . -type f -name "*.json" -o -name "*.go"`

Reply:
Actually don't need .go -- I've removed it from `GENERATION_INPUTS` & the command still runs successfully.